lib/bearerstrategy.js

```diff
@@ -308,7 +308,7 @@ Strategy.prototype.authenticate = function authenticateStrategy(req) {
             .concat(`&${aadutils.getLibraryVersionParameterName()}=${aadutils.getLibraryVersion()}`);
 
         // use the policy name as the metadata cache key
-        cacheKey = 'policy: ' + policyName;
+        cacheKey = 'policy: ' + self._options.policyName;
```

Comment:
Do you mean to include this change? B2C will still be broken even with this change, right?